backend/app/app/services/chat_agent/tools/library/basellm_tool/basellm_tool.py

```diff
@@ -70,8 +71,10 @@ async def _arun(
             "query",
             args[0],
         )
-        # Only use the latest human message
-        query = ToolInputSchema.parse_raw(query).latest_human_message
```

Comment:
Should we use `standard_query_format` as the default string formatting for chat history and tool input for all tools?